scripts/publish-npm.js

```diff
@@ -1,192 +0,0 @@
-const { execSync, execFileSync } = require('child_process');
```

Comment:
This script is referenced from the root `package.json`, I think you’ll want to remove it from there as well